pulumi/go.mod

```diff
@@ -1,9 +1,57 @@
 module github-issue-automation
 
-go 1.16
+go 1.19
 
 require (
 	github.com/pulumi/pulumi-aws/sdk/v4 v4.31.0
 	github.com/pulumi/pulumi/sdk/v3 v3.19.0
```

Comment:
are we able to use a newer pulumi sdk now that we are upgrading go version?

Reply:
Yes - added https://github.com/pulumi/github-issue-automation/issues/33